unittests/allplatformstests.py

```diff
@@ -4692,7 +4692,7 @@ def test_scripts_loaded_modules(self):
         cmd = self.meson_command + ['--internal', 'test_loaded_modules', '--unpickle', str(p)]
         p = subprocess.run(cmd, stdout=subprocess.PIPE)
         all_modules = json.loads(p.stdout.splitlines()[0])
-        meson_modules = [m for m in all_modules if 'meson' in m]
+        meson_modules = [m for m in all_modules if 'meson' in m and not m.startswith('__')]
```

Comment:
I'm not certain I understand what editable installs have to do with anything here.

(It's not even something we officially support, we provide the meson.py launcher script for this.)

Reply:
This is because it add `'__editable___meson_1_0_99_finder'` to the list of loaded modules.

Perharps `[m for m in all_modules if m.startswith('mesonbuild')]` would be better?

Reply:
Sounds good, I like that commit.